Description: Upstream changes introduced in version 2.64-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 ssmtp (2.64-1) unstable; urgency=low
 .
   * New upstream version
     Delete 02-397149-amd64-crammd5.patch (merged)
   * Source package format is 3.0 (quilt)
   * Fix out-of-date-standards-version
   * Fix malformed-prompt-in-templates
   * Fix no-homepage-field
   * Add 02-557725-solaris.patch by Darik Horn to make ssmtp compatible
     with Solaris 11. Closes: 557725
   * Add 03-557741-remote-addr.patch by Victor Sudakov to insert the X-
     Originating-IP header. Closes: 557741
   * Make AuthUser/AuthPass not visible to all users. Closes: 500454
 .
 The person named in the Author field signed this changelog entry.
Author: Anibal Monsalve Salazar <anibal@debian.org>
Bug-Debian: http://bugs.debian.org/500454
Bug-Debian: http://bugs.debian.org/557725
Bug-Debian: http://bugs.debian.org/557741

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- ssmtp-2.64.orig/ssmtp.conf.5
+++ ssmtp-2.64/ssmtp.conf.5
@@ -26,7 +26,7 @@ The user that gets all mail for userids 
 .Pp
 .It Cm Mailhub
 The host to send mail to, in the form
-.Ar host No | Ar IP_addr No Oo : Ar port Oc .
+.Ar host port | Ar IP_addr port Oo : Ar port Oc .
 The default port is 25.
 .Pp
 .It Cm RewriteDomain
